backend/platal/auth/factories.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import datetime
+import django.utils.text
+import django.utils.timezone
+import factory
+import factory.django
+
+from platal.auth import models
+
+
+class AccountFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = models.Account
+
+    firstname = factory.Faker('first_name')
+    lastname = factory.Faker('last_name')
+    full_name = factory.LazyAttribute(lambda o: '{} {}'.format(o.firstname, o.lastname))
+    directory_name = factory.LazyAttribute(lambda o: '{} {}'.format(o.firstname, o.lastname))
+    sort_name = factory.LazyAttribute(lambda o: '{} {}'.format(o.lastname, o.firstname))
+    display_name = factory.LazyAttribute(lambda o: '{} {}'.format(o.firstname, o.lastname))
+    sex = factory.Sequence(lambda n: ('female', 'male')[n % 2])
+
+    @factory.sequence
+    def registration_date(n):
+        return django.utils.timezone.make_aware(
+            datetime.datetime(2000, 1, 1) + datetime.timedelta(days=n),
+            django.utils.timezone.get_fixed_timezone(0))
```

Comment:
Should be simpler using pytz:
```python
return pytz.UTC.localize(datetime.datetime(2000, 1, 1) + datetime.timedelta(days=n))
```

Another option with http://factoryboy.readthedocs.org/en/latest/fuzzy.html#fuzzydatetime:
```python
registration_date = factory.fuzzy.FuzzyDateTime(
    start_dt=datetime.datetime(2000, 1, 1, tzinfo=pytz.UTC),
    end_dt=datetime.datetime.now(tz=pytz.UTC),
)
```